src/core/hle/libraries/libkernel/file_system.h

```diff
@@ -7,11 +7,37 @@ class SymbolsResolver;
 }
 
 namespace Core::Libraries::LibKernel {
+constexpr int SCE_MAX_PATH = 255;
```

Comment:
```suggestion

constexpr int SCE_MAX_PATH = 255;
```
There should be a space after declaration of namespace and before the end of namespace

Reply:
ok